NuGet.config

```diff
@@ -5,17 +5,10 @@
   </solution>
   <packageSources>
     <clear />
-    <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
-    <!--  Begin: Package sources from dotnet-aspnetcore -->
-    <add key="darc-pub-dotnet-aspnetcore-2d82d2c" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-aspnetcore-2d82d2cc/nuget/v3/index.json" />
-    <!--  End: Package sources from dotnet-aspnetcore -->
-    <!--  Begin: Package sources from dotnet-runtime -->
-    <add key="darc-pub-dotnet-runtime-cfb9e77" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-runtime-cfb9e770/nuget/v3/index.json" />
-    <!--  End: Package sources from dotnet-runtime -->
-    <!--End: Package sources managed by Dependency Flow automation. Do not edit the sources above.-->
     <!-- Feeds used in Maestro/Arcade publishing -->
     <add key="dotnet5" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json" />
```

Comment:
do we get to remove the dotnet5 instance yet?

Reply:
I think it's safe to remove now, but I'm thinking this can be removed when we remove .NET 5 all up from this repo.